api/organisations/urls.py

```diff
@@ -145,5 +146,10 @@
             url(r"^", include(organisations_router.urls)),
             url(r"^", include(nested_roles_router.urls)),
             url(r"^", include(nested_user_roles_routes.urls)),
+            path(
+                "api/v1/organisations/<int:organisation_pk>/roles-api-key/<str:prefix>/",
+                RolesbyMasterAPIPrefixViewSet.as_view({"get": "list"}),
+                name="roles_api_keys_by_prefix",
+            ),
```

Comment:
why go with `as_view` here instead of adding to the router?

Reply:
The URL was updated and it was added to the router